build.xml

```diff
@@ -123,18 +123,36 @@
 
         <exec command="composer install" dir="${drush.root}/.." passthru="true" />
 
+        <!-- Swap out the composer file for our git repository -->
```

Comment:
@becw So I'm, cheating here a little, using composer to checkout the drupal release and its dependencies, and then deleting that and replacing it with a symlink to a git checkout.

Reply:
Yep, this cheat makes sense! However, I'm a bit wary of deleting the existing path if it exists -- what if someone re-runs `phing init` later on? For example, the README states:

> If you re-run composer in any of the docroots you may need to re-run `phing init`

It could make sense to fail if the directories exist, or prompt before deleting, or something -- I know this makes the script quite a bit messier, but it could prevent some very frustrating code loss.